src/EnergyPlus/BranchNodeConnections.hh

```diff
@@ -67,6 +68,9 @@ namespace DataBranchNodeConnections {
 
 namespace BranchNodeConnections {
 
+    extern const std::array<std::string_view, static_cast<int>(DataLoopNode::ConnectionObjectType::Num)> ConnectionObjectTypeNames;
+    extern const std::array<std::string_view, static_cast<int>(DataLoopNode::ConnectionObjectType::Num)> ConnectionObjectTypeNamesUC;
```

Comment:
Moved these name arrays here from DataLoopNode (but didn't move the enum yet because that'll touch every file that manages HVAC nodes.